### ISSUE TITLE
Error occurs when using `webpack` loader query string as import in rollup

### ISSUE THREAD

**Issue by roastlechon** [rollup/rollup#1486]
TypeError occurs when using `webpack` loader query string as import in `rollup`.

```
Cannot use ‘in’ operator to search for ‘default’ in /assets/file-loader-imported-filename.9b951a6598d24f97acccd61c43d8df97.js 
```

Our use case is using a `webpack` loader import like:

```
import mosaic from '!!file-loader?publicPath=/assets/&name=file-loader-imported-filename.[hash].[ext]!@external-module';
```

Because of the interopBlock looking for `default` via `in` keyword, it throws if its a string.  To compromise, I tested and created a PR that uses hasOwnProperty instead to ensure both worlds work.

**Comment by paulfalgout** [comment]
The bugfix is released, correct?

**Comment by roastlechon** [comment]
Looks released as of v0.45.1
